executor/runtime/docker/docker.go

```diff
@@ -1054,6 +1054,14 @@ func (r *DockerRuntime) Prepare(ctx context.Context, pod *v1.Pod) error { // nol
 		})
 	}
 
+	if runtimeTypes.GetSidecarConfig(systemServices, runtimeTypes.SidecarTrafficSteering).EnabledCheck(&r.cfg, r.c) {
+		if r.c.TrafficSteeringEnabled() {
+			r.c.SetEnvs(map[string]string{
+				"TITUS_SECCOMP_AGENT_HANDLE_TRAFFIC_STEERING": "true",
```

Comment:
Are you sure this is a new Titus System Service?

This variable kinda sounds like it is handled by the existing TSA

Reply:
Yeah pretty sure its a new TSS :) The systemd file has a new binary 
`
ExecStart=/usr/bin/nise -task_id %i
`

https://docs.google.com/document/d/1E3KnpagrgQDh5rellBU1-hYWf5e6TbQ_z--y6wxN58I/edit#heading=h.n4vq1kaspkzl

TSA is used as well to make the whole solution work.